.github/workflows/client-pr.yml

```diff
@@ -49,11 +50,23 @@ jobs:
             package-manager: yarn
             threshold: 75
             working-directory: ${{ env.working-directory }}
-        
-      - uses: netlify/actions/cli@master 
+
+      - name: Deploy to Netlify
+        id: "deploy"
+        uses: nwtgck/actions-netlify@v1.2
+        with:
+          publish-dir: 'client/build'
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          deploy-message: ${{ github.event.pull_request.title }}
+          alias: ${{ github.head_ref }}
```

Comment:
this makes the deployment url based on the branch name. so for this PR it would be `https://33-domain-deployments--hour-logger.netlify.app/`.